doc/api/url.md

```diff
@@ -515,7 +516,7 @@ const params = new URLSearchParams({
   query: ['first', 'second']
 });
 console.log(params.getAll('query'));
-  // Prints ['first,second']
+  // Prints [ 'first,second' ]
```

Comment:
This is an array with a single element :) It literally prints `[ 'first,second' ]`.